environment.yml

```diff
@@ -0,0 +1,26 @@
+name: py2_parcels
+channels:
+  - https://conda.anaconda.org/conda-forge
+  - https://repo.continuum.io/pkgs/free
+dependencies:
+  - python=2.7
+  - cachetools>=1.0.0
```

Comment:
I've seen `basemap` mentioned a few times. Should it be included as a requirement here?

Reply:
We're not enforcing `Basemap`, because on many HPC systems it's not available.  Parcels still runs without `Basemap` installed; users just won't be able to plot continents etc